packages/codecs/ilp/test/typing.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { assert } from 'chai'
-import { isFulfill, isPrepare, isReject } from '..'
+import { isFulfill, isPrepare, isReject } from '@interledger/codecs-ilp'
```

Comment:
The difference here is this style, with the full explicit name, will work out of the box, without having to yarn build first ... due to the root tsconfig.json having set `paths`